ivy_tests/test_ivy/test_frontends/test_torch/test_pointwise_ops.py

```diff
@@ -1028,3 +1028,126 @@ def test_torch_absolute(
         input=np.asarray(input, dtype=input_dtype),
         out=None,
     )
+
+
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=("bool",), num_arrays=1),
```

Comment:
Hello @ahmedo42 , the dtype should not be limited to only bool as the function can handle other dtypes as well as mentioned in the docs [here](https://pytorch.org/docs/stable/generated/torch.logical_not). Please apply that to the other functions where it is relevant as well. Thanks!